src/main/java/reposense/parser/CsvParser.java

```diff
@@ -16,6 +16,7 @@
 import reposense.system.LogsManager;
 
 public abstract class CsvParser<T> {
+    public static final String DEFAULT_BRANCH = "HEAD";
```

Comment:
Hmm I've been specifically asked to move DEFAULT_BRANCH declaration to CsvParser by my previous reviewer @eugenepeh . This is because the default branch is only used when the branch parameter is empty in the CSV file.

Reply:
>Hmm I've been specifically asked to move DEFAULT_BRANCH declaration to CsvParser by my previous reviewer @eugenepeh . This is because the default branch is only used when the branch parameter is empty in the CSV file.

Yes, however most of my suggestions were based on the current implementation. E.g. if `DEFAULT_BRANCH` is declared in both the subclasses, the most reasonable rationale would be to shift them to the parent class. Our reviews may not always be on the architectural point of view, but sometimes to clean up the unnecessary codes to make the subsequent iterations clearer.

Reply:
@apoorva17 Yes, since we are changing the default branch from `master` to `HEAD`, it should also work for the other use case (using `-repo` command instead). 

`parse_configOrLocationsSimilar_success()` as it still keeps the `master` branch in the config files as it was still mandatory before this PR. Since it is now optional, we can remove that `master` branch setting in the config files, and this test should pass now.

Reply:
```DEFAULT_BRANCH``` has been moved to ```RepoConfiguration``` class and config file has been updated